### DISCUSSION TITLE
Intellisense C C++ showing incomplete documentation

### DISCUSSION THREAD

**Discussion by bolovanos** [microsoft/vscode-cpptools#8428] [Q&A]
Hi,
is there a way to show more of a documentation. For example on printf I get
![image](https://user-images.githubusercontent.com/12034422/142736625-2dd23aa7-fdac-4fdf-b0ad-c07df2a43700.png)

I would rather want something more of a (man 3 printf) or what pycharm community has
![image](https://user-images.githubusercontent.com/12034422/142736639-7e3d6b8c-32a3-4a65-b5cc-d40f0d4ff59a.png)

How is docu linked to intellisense? This what I got looks like synopsis from glibc only.
Can I change it somehow?

Thank you.

-------------- Info
plugin C C++ v1.7.1
Visual studio code 1.62.2
Win 10 64 bit
gcc version 6.3.0 (MinGW.org GCC-6.3.0-1)

**Comment by sean-mcmanus** [comment]
We just parse the header file for documentation comments. The publishers of the headers would need to add comments.
